extensions/notebook/src/common/notebookUtils.ts

```diff
@@ -3,7 +3,18 @@
  *  Licensed under the Source EULA. See License.txt in the project root for license information.
```

Comment:
this would be the 3rd notebookUtils file that we have in the notebooks space. Can we come up with another name to disambiguate? Just anything else please for our sanity 😄